datasets/multi_woz_v22/multi_woz_v22.py

```diff
@@ -0,0 +1,282 @@
+               Srinivas Sunkara and
+               Raghav Gupta and
+               Jianguo Zhang and
+               Jindong Chen},
+  title     = {MultiWOZ 2.2 : {A} Dialogue Dataset with Additional Annotation Corrections
+               and State Tracking Baselines},
+  journal   = {CoRR},
+  volume    = {abs/2007.12720},
+  year      = {2020},
+  url       = {https://arxiv.org/abs/2007.12720},
+  archivePrefix = {arXiv},
+  eprint    = {2007.12720}
+}
+"""
+
+# TODO: Add description of the dataset here
+# You can copy an official description
+_DESCRIPTION = """\
+Multi-Domain Wizard-of-Oz dataset (MultiWOZ), a fully-labeled collection of human-human written conversations spanning over multiple domains and topics.
+MultiWOZ 2.1 (Eric et al., 2019) identified and fixed many erroneous annotations and user utterances in the original version, resulting in an
+improved version of the dataset. MultiWOZ 2.2 is a yet another improved version of this dataset, which identifies and fizes dialogue state annotation errors
+across 17.3% of the utterances on top of MultiWOZ 2.1 and redefines the ontology by disallowing vocabularies of slots with a large number of possible values
+(e.g., restaurant name, time of booking) and introducing standardized slot span annotations for these slots.
+"""
+
+_LICENSE = "Apache License 2.0"
+
+# TODO: Add link to the official dataset URLs here
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URL_LIST = [
+    ("dialogue_acts", "https://github.com/budzianowski/multiwoz/raw/master/data/MultiWOZ_2.2/dialog_acts.json")
+]
+_URL_LIST += [
+    (
+        f"train_{i:03d}",
+        f"https://github.com/budzianowski/multiwoz/raw/master/data/MultiWOZ_2.2/train/dialogues_{i:03d}.json",
+    )
+    for i in range(1, 18)
+]
+_URL_LIST += [
+    (
+        f"dev_{i:03d}",
+        f"https://github.com/budzianowski/multiwoz/raw/master/data/MultiWOZ_2.2/dev/dialogues_{i:03d}.json",
+    )
+    for i in range(1, 3)
+]
+_URL_LIST += [
+    (
+        f"test_{i:03d}",
+        f"https://github.com/budzianowski/multiwoz/raw/master/data/MultiWOZ_2.2/test/dialogues_{i:03d}.json",
+    )
+    for i in range(1, 3)
+]
+
+_URLs = dict(_URL_LIST)
+
+
+class MultiWozV22(datasets.GeneratorBasedBuilder):
+
+    VERSION = datasets.Version("2.2.0")
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(name="v2.2", version=datasets.Version("2.2.0"), description="MultiWOZ v2.2"),
+    ]
+
+    DEFAULT_CONFIG_NAME = "v2.2"  # It's not mandatory to have a default configuration. Just use one if it make sense.
+
+    def _info(self):
+        features = datasets.Features(
+            {
+                "dialogue_id": datasets.Value("string"),
+                "services": datasets.Sequence(datasets.Value("string")),
+                "turns": datasets.Sequence(
+                    datasets.Features(
```

Comment:
No need to instantiate `Features` again. Directly using a dictionary does the job.